relay-general/src/protocol/event.rs

```diff
@@ -468,6 +468,11 @@ pub struct Event {
     #[metastructure(omit_from_schema)] // we only document error events for now
     pub spans: Annotated<Array<Span>>,
 
+    /// Measurements
```

Comment:
Can we add a little more description of what measurements are? Usual format would be:

```
/// One-liner describing the field.
///
/// More information in multiple paragraphs, can be markdown formatted.
```

Reply:
Done in https://github.com/getsentry/relay/pull/724/commits/d0cb881dab6d4aae1b74cb579f2aa71881e04cba